app/views/reviews/index.html.erb

```diff
@@ -4,7 +4,7 @@
   </div>
 
   <div class="panel-body">
-    <div class="star-rating" data-score= <%= @average_review %> ></div>
+    <div class="star-rating" data-score= <%= @product.average_rating %> ></div>
```

Comment:
I'm surprised this does't need quotes around it, like `<div class="star-rating" data-score="<%= @product.average_rating %>" ></div>`. Learn something new erryday :)